sequencer/src/bin/deploy.rs

```diff
@@ -313,7 +342,7 @@ async fn deploy_light_client_contract<M: Middleware + 'static>(
         .context("error linking PlonkVerifier lib")?;
     bytecode
         .link_fully_qualified(
-            "contracts/src/libraries/LightClientStateUpdateVK.sol:LightClientStateUpdateVK",
+            "contracts/tests/mocks/LightClientStateUpdateVK.sol:LightClientStateUpdateVK",
```

Comment:
yeah, a bit confused why are we using `/mocks/` inside `deploy_production_()` function